app/steps/ui/will/codicilsnumber/index.js

```diff
@@ -16,6 +16,11 @@ class CodicilsNumber extends ValidationStep {
         return ctx;
     }
 
+    handlePost(ctx, errors) {
+        ctx.codicilsNumber = ctx.codicilsNumber || 0;
+        return [ctx, errors];
+    }
```

Comment:
Can we please have unit tests for this method, we'll need one for when `ctx.codicilsNumber` exists and one for when `ctx.codicilsNumber` doesn't exist.

Reply:
Argh, doing 3 things at the same time on a Friday afternoon doesn't help :)
Added.